src/ssl.c

```diff
@@ -34433,6 +34480,117 @@ void *wolfSSL_OPENSSL_malloc(size_t a)
     return (void *)XMALLOC(a, NULL, DYNAMIC_TYPE_OPENSSL);
 }
 
+int wolfSSL_OPENSSL_hexchar2int(unsigned char c)
+{
+    int ret = -1;
+
+    switch (c) {
```

Comment:
Do a range check instead of case statement.

Reply:
Nice, that's a lot cleaner/shorter.